Tensile/Source/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ set(TENSILE_USE_OPENMP   ON CACHE BOOL "Use OpenMP to improve performance.")
 set(TENSILE_STATIC_ONLY  ON CACHE BOOL "Disable exposing Tensile symbols in a shared library.")
 
 if(NOT DEFINED CXX_VERSION_STRING)
-    if(CMAKE_CXX_COMPILER MATCHES ".*/hipcc$")
+if(CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" OR CMAKE_CXX_COMPILER MATCHES ".*clang\\+\\+")
```

Comment:
Does this check match version-suffixed compiler names? If it needs that `.*` at the start to match `amdclang++`, I think it might need a `.*` on the end to match `clang++-17`.

Reply:
I am exploring removing this logic altogether in favor of `CMAKE_<LANG>_COMPILER_VERSION`